file/nifile.yml

```diff
@@ -146,6 +146,10 @@ definitions:
         type: integer
         format: int64
         example: 7277
+      workspace:
```

Comment:
It seems strange that the 19.6 file doc is talking about workspaces. I'm not familiar with why that field was added in 19.6, but I just thought I'd point this out.

Reply:
Yeah, it appears to have been submitted to master relatively early in the release. We shipped 19.6 with this in the doc.